_posts/2020-09-22-RS-first-class-citizen.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+image: '/img/Class_159_First_Class_Seating.jpg'
+title: "How to make research software a first-class citizen in the Netherlands?"

```

Comment:
Why the questionmark?

Reply:
I guess it is a question 'the reader' is asking to himself 
Q: How do I make the world better?
A: By following these recommendations.

But we can remove the question mark if you feel it doesn't make sense.

Reply:
No problem, we can also leave it like this? ;-)